tests/components/waze_travel_time/test_config_flow.py

```diff
@@ -154,3 +155,46 @@ async def test_invalid_config_entry(hass, caplog):
     assert result2["errors"] == {"base": "cannot_connect"}
 
     assert "Error trying to validate entry" in caplog.text
+
+
+@pytest.mark.usefixtures("mock_update")
+async def test_reset_filters(hass: HomeAssistant) -> None:
+    """Test resetting inclusive and exclusive filters to empty string."""
+    options = {**DEFAULT_OPTIONS}
+    options[CONF_INCL_FILTER] = "test"
+    options[CONF_EXCL_FILTER] = "test"
+    config_entry = MockConfigEntry(
+        domain=DOMAIN, data=MOCK_CONFIG, options=DEFAULT_OPTIONS, entry_id="test"
```

Comment:
Did you mean `options` here?
```suggestion
        domain=DOMAIN, data=MOCK_CONFIG, options=options, entry_id="test"
```

Reply:
Yep. thank you!